### ISSUE TITLE
Unable to change usernames when using Casehandling

### ISSUE THREAD

**Issue by edwinvanelk** [mendix/LoginForm#12]
When Casehandling is switched on, cursor is placed at the end of the username after every cursor movement or after every inserted character.

Trying to use the arrow keys to change (characters in) the username is impossible. Even selecting a single character using the mouse does not work.

**Comment by mgroeneweg** [comment]
Just about to report the same thing! Any update here?

**Comment by edwinmp** [comment]
This issue is caused by the keyup event:

```
on(this._userInput, "keyup", lang.hitch(this, function(e) {
     this._userInput.value = this._userInput.value.toUpperCase();
 }));
```
Found at:
https://github.com/mendix/LoginForm/blob/master/src/LoginForm/widget/LoginForm.js#L216

**Solution:** Remove the keyup event. There's already a text-transform property being used to handle the case options, though its effects are only on the client side.
To ensure that the transformation is passed to the server, the code to change the value to upper or lowercase should be done just before the Username is send to the server.

Will have this implemented in the next pull request.

**Comment by JelteMX** [comment]
Fixed in [3.5.0](https://github.com/mendix/LoginForm/releases/tag/v3.5.0)
